### PR TITLE
Improve collector descriptions

### PR DESCRIPTION
**Pull request by marga-kinvolk**
"Collect processes" and "Collect sockets" doesn't really help clarify what these gadgets do. Reword this to "Gather information" about either running processes or network sockets, so that we actually provide some additional info in the short description.

### CODE REVIEW

**Comment by alban** [comment]
Could you also improve the messages at:

https://github.com/kinvolk/inspektor-gadget/blob/main/pkg/gadgets/process-collector/gadget.go#L35
https://github.com/kinvolk/inspektor-gadget/blob/main/pkg/gadgets/process-collector/gadget.go#L44
https://github.com/kinvolk/inspektor-gadget/blob/main/pkg/gadgets/socket-collector/gadget.go#L42
https://github.com/kinvolk/inspektor-gadget/blob/main/pkg/gadgets/socket-collector/gadget.go#L54

and run `make generate-documentation`? Thanks

**Review by blanquicet — APPROVED** [review]
LGTM

File: pkg/gadgets/socket-collector/gadget.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ func (f *TraceFactory) Operations() map[string]gadgets.TraceOperation {
 
 	return map[string]gadgets.TraceOperation{
 		"start": {
-			Doc: "Collect a snapshot of the list of sockets",
+			Doc: "Create a snapshot of the active TCP and UDP sockets",
```
Comment (alban) [inline pkg/gadgets/socket-collector/gadget.go:54]: It also show so-called passive TCP sockets (TCP sockets in the LISTEN state)
```suggestion
			Doc: "Create a snapshot of the TCP and UDP sockets",
```

Comment (marga-kinvolk) [reply pkg/gadgets/socket-collector/gadget.go:54]: Yeah, ok. Active was the wrong word and I knew this, but I wanted to highlight that they were the current ones. But "current sockets" also sounds wrong. :-/ Would "currently open sockets" be the right expression?

Comment (alban) [reply pkg/gadgets/socket-collector/gadget.go:54]: I thought that "snapshot" would convey that. What about:

> "Take a snapshot of the currently open TCP and UDP sockets. Once taken, the snapshot is not updated automatically. However one can call the `start` operation again at any time to update the snapshot."

Comment (marga-kinvolk) [reply pkg/gadgets/socket-collector/gadget.go]: I didn't know this could be a longer string.  Alright, I've taken that and added the same sentence to the process one. PTAL?

**Comment by blanquicet** [comment]
I think this PR should also update the ['How to use'](https://github.com/kinvolk/inspektor-gadget/blob/main/README.md#how-to-use) given that #271 was already merged.

File: pkg/gadgets/socket-collector/gadget.go (hunk 1 above)
Comment (alban) [reply pkg/gadgets/socket-collector/gadget.go]: LGTM!

**Review by alban — APPROVED** [review]
LGTM
